installation/on_alpine_linux.md

```diff
@@ -0,0 +1,12 @@
+# On Alpine Linux
+
+Alpine Linux includes the Crystal compiler in the community repository, starting from version Alpine 3.7.
+
+Currently, the package is only available for `aarch64` and `x86_64` platforms.
+
+## Install
+
+```bash
+sudo apk update
+sudo apk add crystal@community shards@community
```

Comment:
I'm pretty sure you don't need the `@community`

Reply:
Me too, but it failed to work without it.

Reply:
It works fine for me

Reply:
Oh, I figured it out. The vagrant box I used (`generic/alpine38`) has `@community` pinned in `/etc/apk/repositories`. That's why it was needed.